src/main/kotlin/uk/gov/justice/digital/hmpps/sentenceplan/data/Note.kt

```diff
@@ -0,0 +1,15 @@
+package uk.gov.justice.digital.hmpps.sentenceplan.data
+
+import java.time.LocalDateTime
+
+data class Note(
+  val noteObject: String,
+  val note: String,
+  val additionalNote: String?,
```

Comment:
Mind explaining this a little? a note can have an additional note?

Reply:
A Plan can be Agreed in one of 3 states. All of them support an optional note field, and two of them have a mandatory note field. "Additional" note here is the "optional" note - we can rename this if it makes it clearer.